controllers/cloud.redhat.com/providers/metrics/utils.go

```diff
@@ -0,0 +1,55 @@
+package metrics
+
+import (
+	crd "cloud.redhat.com/clowder/v2/apis/cloud.redhat.com/v1alpha1"
+	deployProvider "cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/providers/deployment"
+	webProvider "cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/providers/web"
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (m *metricsProvider) makeMetrics(deployment *crd.Deployment, app *crd.ClowdApp) error {
```

Comment:
does this need to be in a separate file from the Provide fn?  It doesn't seem to be used anywhere else.

Reply:
it was for consistency and future expansion